source/_patterns/02-organisms/sections/comments.mustache

```diff
@@ -0,0 +1,11 @@
+{{# comments }}
+<section class="comments spacing padding-bottom">
+  <h3 class="font--tertiary--l theme--primary-text-color">3 comments</h3>
+  <hr>
+  {{# comment }}
+    {{> molecules-comment }}
+  {{/ comment }}
+  <hr>
+  {{> molecules-comment-form }}
+</section>
+{{/ comments }}
```

Comment:
This is a great start! It looks like the basics to a comment thread/form is there. I left a comment above about the additional variations of the comments.

As for the design, I feel like there's a little more room to make this more visually appealing. 

Here's a few ideas:
- Increase the avatar size by about 50% and make it round
- add a `<hr>` between each comment in the thread (unless it's the last one)
- move the timestamp below the comment and make it smaller and a more subtle gray color
- add a little more space between the commenter's name, the comment body, and the timestamp
- remove the word "Comment" above the textarea. i think people will know that's where the comment goes
- remove the "Logged in as..." part. i don't think that's necessary

And if there's anything else you think might work to make this more appealing, please feel free to add it in.

![img](http://d.pr/i/5And+)